sample/src/main/java/kin/sdk/core/sample/kin/sdk/core/sample/dialog/ConfirmDialog.java

```diff
@@ -0,0 +1,29 @@
+package kin.sdk.core.sample.kin.sdk.core.sample.dialog;
+
+import android.content.Context;
+
+public class ConfirmDialog extends ErrorDialog {
+
+    public interface OnConfirmListener {
+
+        void onConfirm();
+    }
+
+    private OnConfirmListener confirmListener;
+
+    public ConfirmDialog(Context context, String message, String confirmationText,
+        OnConfirmListener confirmListener) {
```

Comment:
maybe we can create a setConfirmationText(String text) and remove String confirmationText  from the constructor.
with a default test "OK"/"Confirm"